org.laptop.TurtleArtActivity.json

```diff
@@ -24,6 +24,7 @@
         "--filesystem=home"
     ],
     "cleanup": [
+	"/app/lib/python3.9/site-packages/pandas/tests",
```

Comment:
Pandas is not being installed so I think this is unneeded

Reply:
I forgot this one!  Thanks for the catch!